api/src/main/java/org/pmiops/workbench/mandrill/MandrillService.java

```diff
@@ -0,0 +1,14 @@
+package org.pmiops.workbench.mandrill;
+
+import org.pmiops.workbench.mandrill.model.MandrillMessage;
+import org.pmiops.workbench.mandrill.model.MandrillMessageStatuses;
+
+/**
+ * Encapsulate mandrill API interaction details and provide a simple/mockable interface
+ * for internal use.
+ */
+public interface MandrillService {
+
+  MandrillMessageStatuses sendEmail(final MandrillMessage m);
+
+}
```

Comment:
What I would prefer is a more generic interface that hides the implementation details ... and then has a more specific implementation. For example, this is really just a `MailService` class. I would suggest that this interface have a `sendPasswordEmail()` method with minimal arguments, and then the impl class can do the message construction in a mandrill-specific way. This way we have a cleaner separation of concerns. The `DirectoryServiceImpl` class above should not have to know about Mandrill details to do it's primary work. As a caller of the `MailService`, the directory service should just ask it to do what it needs to do, however it wants to do it. The way this is written now (which is also what it was when sending a plain `Message` object), different callers need to have specialized knowledge that they really don't *need* to have and is better off encapsulated in the impl class.